server/controllers/post/index.js

```diff
@@ -0,0 +1,11 @@
+const create = require('./create');
+const getPostById = require('./getPostById');
+const updatePostById = require('./updatePostById');
+const deleteById = require('./deleteById');
+
+module.exports = {
+	...create,
+	...getPostById,
+	...updatePostById,
+	...deleteById
```

Comment:
On requiring each module well, they are accessible using `create.create` like object inside an object so to avoid doing that I used the spread operator.